examples/notebooks/quickstart.md

```diff
@@ -0,0 +1,99 @@
+---
+jupyter:
+  jupytext:
+    cell_metadata_filter: -all
+    text_representation:
+      extension: .md
+      format_name: markdown
+      format_version: '1.3'
+      jupytext_version: 1.14.0
+  kernelspec:
+    display_name: Bash
+    language: bash
+    name: bash
+---
+
+# Quickstart
+
+This notebook showcases the key use cases of VAST in an interactive manner.
+
+:::caution Work in Progress
+This page is work in progress. It is the first example of a [notebook-based
+approach of writing user
+guides](/docs/develop-vast/contribution-workflow/notebooks), but the content and
+CI-workflow is not fully fleshed out. Stay tuned.
+:::
+
+## Install VAST
+
+First, let's get a VAST binary to play with. The easiest way to [setup
+VAST](/docs/setup-vast) is downloading a static binary on Linux.
+
+:::note Coming soon: the VAST installer
+The easiest way to obain VAST is through our installer script:
+
+```bash .noeval
+/bin/bash -c "$(curl -fsSL https://vast.io/install.sh)"
+```
+
+On Linux, the script downloads a static build. On macOS, the script clones the
+repo and creates a release build.
+:::
+
+Following the instructions to add `/opt/vast/bin` to your `PATH`, test
+whether you are ready to use VAST:
+
+```bash .noeval
+vast version
+```
+
+## Start a VAST node
+
+Begin with starting a VAST node:
+
+```bash .noeval
+vast start
+```
+```
+[14:12:09.207] VAST v2.1.0 is listening on localhost:42000
+```
```

Comment:
This is weird, I thought the outputs were not recorded in the original notebooks

Reply:
ah, you manually pasted that in because all cells are tagged as `.noeval` as you don't have VAST running in the background, correct?

Reply:
Yeah, I'm still cheating.